### DISCUSSION TITLE
Plex LXC + Boot disk size

### DISCUSSION THREAD

**Discussion by MoldyTaint** [tteck/Proxmox#1589] [General]
Hello! I know i'm always a pain asking all sorts of oddball questions. 

I understand Plex LXC will grow in size. Right now it's at 10GB with 8GB used after I just bumped it from 6GB to 10GB. Is it normal for it to go up that much in a night though? Maybe i'm just confused and over thining things. I do have a few screenshots to maybe help clear my head a bit.

![image](https://github.com/tteck/Proxmox/assets/98976440/c6b96854-0872-433b-a621-ec5d54e41a50)

![image](https://github.com/tteck/Proxmox/assets/98976440/6e3ddd18-8151-4603-8b86-51fe7e322b52)

**Comment by oOStroudyOo** [comment]
Plex pretty much sums it up here: https://support.plex.tv/articles/202529153-why-is-my-plex-media-server-directory-so-large/

**Reply by oOStroudyOo** [reply]
You can turn off what stuff gets cached.
So be flexible with what you prefer, faster loading times for the sacrifice of some storage, I'll take that.

I'm using 15GB of diskstorage but that's saving the cache from 27k songs, 4200 TV episodes, and 300 films. not bad if you ask me.
